src/service/okapi.service.ts

```diff
@@ -10,7 +10,7 @@ class OkapiService extends RestService {
 
     let okapi_login_path = config.get('okapi_login_path');
     if (okapi_login_path === undefined) {
-      okapi_login_path = '/authn/login';
+      okapi_login_path = '/bl-users/login-with-expiry?expandPermissions=true&fullPermissions=true';
```

Comment:
We are planning on doing an RTR Upgrade Project in 2025, after the Poppy Upgrade.

Reply:
We need an option to designate which login end point to use.

Reply:
```
fw config set okapi_login_path /bl-users/login
```